synapse/lib/module.py

```diff
@@ -96,7 +96,7 @@ def getModPath(self, *paths):
         dirn = self.getModDir()
         return s_common.genpath(dirn, *paths)
 
-    def initCoreModule(self):
+    async def initCoreModule(self):
```

Comment:
Since this is now a `async` function, do module implementers have to be aware of anything special? if so, that shoudl be noted in the docstring.

Reply:
ok